docs/guidance/gcp-gke-gpu-cluster.md

```diff
@@ -0,0 +1,76 @@
+# Start Google Cloud GKE Cluster with GPUs for KubeRay
+
+## Step 1: Create a Kubernetes cluster on GKE
+
+Run this command and all following commands on your local machine or on the [Google Cloud Shell](https://cloud.google.com/shell). If running from your local machine, you will need to install the [Google Cloud SDK](https://cloud.google.com/sdk/docs/install).
+
+```sh
+gcloud container clusters create kuberay-gpu-cluster \
+    --num-nodes=1 --min-nodes 0 --max-nodes 1 --enable-autoscaling \
+    --zone=us-west1-b --machine-type e2-standard-4
+```
+
+This command creates a Kubernetes cluster named `kuberay-gpu-cluster` with 1 CPU node in the `us-west1-b` zone. In this example, we use the `e2-standard-4` machine type, which has 4 vCPUs and 16 GB RAM.
```

Comment:
It might be clearer if we move this sentence above the snippet above

Reply:
Updated https://github.com/ray-project/kuberay/pull/1223/commits/1eca0ec66b8be589cb2304d56e4fd8451a286ff3